src/libs/actions/Policy/Policy.ts

```diff
@@ -1814,9 +1814,13 @@ function buildPolicyData(
     const optimisticMccGroupData = buildOptimisticMccGroup();
 
     const shouldEnableWorkflowsByDefault =
-        !introSelected?.choice || introSelected.choice === CONST.ONBOARDING_CHOICES.MANAGE_TEAM || introSelected.choice === CONST.ONBOARDING_CHOICES.LOOKING_AROUND;
+        !introSelected?.choice ||
+        introSelected.choice === CONST.ONBOARDING_CHOICES.MANAGE_TEAM ||
+        introSelected.choice === CONST.ONBOARDING_CHOICES.LOOKING_AROUND ||
+        introSelected.choice === CONST.ONBOARDING_CHOICES.PERSONAL_SPEND;
```

Comment:
```suggestion
        introSelected.choice === CONST.ONBOARDING_CHOICES.TRACK_WORKSPACE;
```

I am just trying to refer this [here](https://github.com/Expensify/App/pull/62239#issuecomment-2902016989)

Reply:
My bad, Track and budget expenses option is PERSONAL_SPEND only..

Reply:
hmm I'm unable to reproduce

https://github.com/user-attachments/assets/9a49db78-2852-4b48-a95e-0cd22593c017

Reply:
Oh, just to confirm I am testing it without offline. As BE is in production it needs to be worked for online too, right?

Reply:
Yea, I just toggled it offline because I couldn't see any issues while online and from your video it seemed to be an optimistic update, since the button changed to `Submit` after a bit (presumably when the API response came in). I'm also unable to reproduce it while online.


https://github.com/user-attachments/assets/f8deb80a-56e6-42af-8cad-b0feb859677f

Reply:
Ah, ok. Let me try with a new account again

Reply:
Looks good now while testing with other account.